controllers/goharbor/harborcluster/cache/redis.go

```diff
@@ -1,13 +1,21 @@
 package cache
 
 import (
-	goharborv1alpha2 "github.com/goharbor/harbor-operator/apis/goharbor.io/v1alpha2"
+	"github.com/goharbor/harbor-operator/apis/goharbor.io/v1alpha2"
 	"github.com/goharbor/harbor-operator/pkg/lcm"
 )
 
 type RedisReconciler struct {
```

Comment:
It seems we don't need such a cache reconciler here. The harborcluster reconcile should call the cache lcm controller to do the harborcluster reconcile process.